specs/mgt-taxonomy-picker.md

```diff
@@ -0,0 +1,87 @@
+# mgt-taxonomy-picker
+
+## Overview
+The taxonomy picker is a component that can query the Microsoft Graph API for Taxonomy and render a dropdown control allowing selection of **a single** term based on the specified term set id and a combination of the specified term set id and the specified term id. 
```

Comment:
Could you add the links to the documentation for this API? I got https://learn.microsoft.com/en-us/graph/api/resources/termstore-term?view=graph-rest-1.0. Is it the correct one?

Reply:
Sure, I will add that link.